src/app/dapp.ts

```diff
@@ -503,7 +503,7 @@ export async function startDApp(dappEns: string, container = document.body, useD
               vault: window.localStorage['evan-vault'],
             },
             // ensure to only load iframes from ipfs
-            utils.devMode.indexOf(dappEns.replace(`.${utils.getDomainName()}`, '')) !== -1
+            (utils.devMode && utils.devMode.indexOf(dappEns.replace(`.${utils.getDomainName()}`, '')) !== -1)
```

Comment:
perfect use case for ts optional chaining and es6 includes

```suggestion
            (utils.devMode?.includes(dappEns.replace(`.${utils.getDomainName()}`, '')))
```

Reply:
awesome! :D